test/Sniffs/Commenting/TestAsset/FileLevelDocBlockDeprecatedLinkTag.fixed.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-coding-standard for the canonical source repository
+ * @copyright https://github.com/zendframework/zend-coding-standard/blob/master/COPYING.md Copyright
```

Comment:
`s/COPYING/COPYRIGHT/` and so on. I think you can likely grep across the code base for these and fix them.